onnxruntime/core/providers/partitioning_utils.cc

```diff
@@ -104,7 +105,8 @@ std::vector<std::vector<const Node*>> CreateSupportedPartitionNodeGroups(
   std::deque<const Node*> nodes_to_process_with_next_group{};
 
   // initialize in-degrees and find root nodes
-  for (const auto& node : graph_viewer.Nodes()) {
+  for (const auto& node_index : graph_viewer.GetNodesInTopologicalOrder()) {
```

Comment:
>GetNodesInTopologicalOrder [](http://example.com/codeflow?start=45&length=26)

just curious about this change - I believe any ordering is fine here as this function will do its own topological sort

Reply:
Changed it back ... I added it when I was debugging an old issue...